tests/purs/bundle/ObjectShorthand.js

```diff
@@ -1,4 +1,4 @@
-var foo = 1;
+export var foo = 1;
```

Comment:
Why was this change necessary?

Reply:
i think it gets removed due to optimization

Reply:
Huh, that doesn't sound good. But this is in the bundler, so 👌

Reply:
yes that seems to me like a bug in the bundler, but I’m not sure